packages/edit-site/src/components/global-styles/use-global-styles-output.js

```diff
@@ -270,7 +292,11 @@ export function getStylesDeclarations(
 		const cssProperty = rule.key.startsWith( '--' )
 			? rule.key
 			: kebabCase( rule.key );
-		output.push( `${ cssProperty }: ${ compileStyleValue( rule.value ) }` );
+		let ruleValue = rule.value;
+		if ( typeof ruleValue !== 'string' ) {
+			ruleValue = resolveRefValue( rule.value, tree );
```

Comment:
This may resolve to an `object` (no ref found on object) or a string value (ref was found and resolved). Is there a use case where we have an object with something other than a ref?

Relates to other comments about simplify the utility function here.

Reply:
It can be an object in cases like padding where you have `{ "top": "0px, ... }` but I think those are covered right now [here](https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-site/src/components/global-styles/use-global-styles-output.js#L223), I don't know if there might be cases right now that require that check, but maybe in the future?